src/app.js

```diff
@@ -25,11 +57,9 @@ export const start = async args => {
 
   const username = program.user;
 
-  let reposList;
-
-  //TODO: prompt for username if none provided
```

Comment:
We haven't completed this TODO, we should leave it in. It's not in the scope of this issue

Reply:
Removing this comment wasn't in the scope of this ticket, let's leave it in